patches/chrome-android-BUILD.gn.patch

```diff
@@ -43,6 +43,15 @@ index f0c7c5c7f7ebb088375e41c2292071cb734d823b..af14f2c5fa4ee6c466337efa2fec498e
  }
  
  android_library("bundle_canary_java") {
+@@ -1137,7 +1141,7 @@ jinja_template_resources("chrome_public_apk_template_resources") {
+     "java/res_template/xml/file_paths.xml",
+     "java/res_template/xml/launchershortcuts.xml",
+     "java/res_template/xml/searchable.xml",
+-    "java/res_template/xml/syncadapter.xml",
++    #"java/res_template/xml/syncadapter.xml",
```

Comment:
I thought this wasn't actually needed when we went through this before? Or can we just put an empty xml file in its place in brave?

Reply:
otherwise this should be removed with a new line `resources -= [ "java/res_template/xml/syncadapter.xml" ]`

Reply:
@bridiver 
>I thought this wasn't actually needed when we went through this before?

At previous attempt to fix https://github.com/brave/brave-core/pull/2823/files we kept `"java/res_template/xml/syncadapter.xml"` entry because we didn't remove entire `<!-- Sync adapter for browser invalidation. -->` section , but just put own Brave service at https://github.com/brave/brave-core/pull/2823/files#diff-2072cb0b7edeb2304b9f449241cb5870R10

>Or can we just put an empty xml file in its place in brave?

This works :+1: 
With overwriting file https://github.com/brave/brave-core/pull/4329/files#diff-0fc79f2212cd200a1221c1320a3efe51R1
I don't see any notifications for account. Requires https://github.com/brave/brave-browser/pull/7736 , but then no need to patch anything for this at all.

Tested on both my phones.

>resources -= [...]

thanks, will use on future patches instead of commenting out, if required